### DISCUSSION TITLE
How to upload a full directory in sftp

### DISCUSSION THREAD

**Discussion by linboqiao** [Eugeny/tabby#5656] [Q&A]
How to upload a full directory in sftp?

**Comment by Alisonispig** [comment]
+1

**Comment by innoners** [comment]
You have to make a folder first with permission 777 

Example
```
mkdir publicfolder
chmod -R 777 publicfolder
```

Then you can now upload via FTP of tabby and then mv it to your desired folder.
Also, you have to zip your folder before you upload.

**Reply by pgkk** [reply]
Not a good way.
